app/_how-tos/kic-get-started-services-routes.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Services and Routes
+description: |
+  Use the a HTTPRoute or Ingress to configure a Service and a Route.
+content_type: how_to
+
+permalink: /kubernetes-ingress-controller/get-started/services-and-routes/
+breadcrumbs:
+  - /kubernetes-ingress-controller/
+  - index: kubernetes-ingress-controller
+    section: Get Started
+
+series:
+  id: kic-get-started
+  position: 2
+
+tldr:
+  q: How do I create a Service and a Route with {{ site.kic_product_name }}
+  a: |
+    Create a HTTPRoute that points to a Kubernetes service in your cluster.
+
+products:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+
+prereqs:
+  skip_product: true
+---
+
+## How Kubernetes resources map to Kong Entities
+
+A Service inside Kubernetes is a way to abstract an application that is running on a set of Pods. This maps to two objects in Kong: Service and Upstream.
+
+The service object in Kong holds the information of the protocol to use to talk to the upstream service and various other protocol specific settings. The Upstream object defines load-balancing and health-checking behavior.
+
+<!--vale off-->
+{% mermaid %}
+flowchart LR
+    H(Request traffic)
+    subgraph Pods
+        direction LR
+        E(Target)
+        F(Target)
+        G(Target)
+    end
+
+    subgraph Kubernetes Service
+        direction TB
+        C(Service)
+        D(Upstream)
+    end
+    
+    subgraph Ingress / HTTPRoute
+        direction LR
+        A(Route)
+        B(Route)
+    end
+
+    A --> C
+    B --> C
+    C --> D
+    D --> E
+    D --> F
+    D --> G
+    H --> A
+
+    linkStyle 6 stroke:#b6d7a8
+{% endmermaid %}
+<!--vale on-->
+
+Routes are configured using Gateway API or Ingress resources, such as `HTTPRoute`, `TCPRoute`, `GRPCRoute`, `Ingress` and more.
+
+## Deploy an echo service
+
+In this guide, you will deploy an `echo` service which returns information about the Kubernetes cluster and route traffic to the service.
+
+```bash
+kubectl apply -f {{ site.links.web }}/manifests/kic/echo-service.yaml -n kong
+```
+
+## Create a HTTPRoute / Ingress
+
```

Comment:
This needs a description. Why are we creating this route? What is it going to do?